CHANGELOG.md

```diff
@@ -1,3 +1,42 @@
+## Release 0.6.0
+
+New features:
+- Preliminary work to add capabilities to nodes, so that the UI will
+  be able to send actions down to the probe.
+  [#315](https://github.com/weaveworks/scope/issues/315)
+- Added support for pcap based packet sniffing, to provide bandwidth
+  usage information.
+  [#317](https://github.com/weaveworks/scope/pull/317)
+- Probes now push data to the app, instead of the app requesting it.
```

Comment:
" instead of the app requesting it." -> " instead of the app pulling it."

Also, whats the user visible change associated with this?